### PR TITLE
fix: Styling alert component with tokens

### PR DESCRIPTION
**Pull request by magnuh**
Known issues / open questions: 

- We need classes for styling color of individual borders, like `border-l` to be able to do: `i-border-l-$color-alert-success-border`. (Did this through the style attribute for now, as in Fabric/react.)
- The automatic class factory doesn't work when trying to dynamically create the classes like using a prop in react for setting the type `i-border-$color-alert-${type}-subtle-border`...but it DO work, however, once I have used the class name hard coded before doing it dynamically, like `i-border-$color-alert-success-subtle-border`. Guessing that the css compiler has already run when the class gets set in react... 🤷 (This thing was driving me crazy, when some things worked and some not. 😅🙈)
- Still left to decide how the css theme file should be included... I just added it as a `<link>` in `<head>` for now.
- The non-dynamic class list of this component should go into the component-classes repo, right?
- The order of the different parts of the token seems somewhat messed up for the alert component (or the others 😅): Should it be `w-alert-color` or `w-color-alert`? (button is `w-button-color`)
- The icons should be themeable as well, right?

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -17,6 +17,7 @@
         visibility: visible;
       }
     </style>
+    <link rel="stylesheet" href="https://assets.finn.no/pkg/@warp-ds/tokens/1.0.0-alpha.9/finn-no.css" />
```
Comment (pearofducks) [inline index.html:20]: This should use an Eik alias so it doesn't need constant updates

**Comment by pearofducks** [comment]
> The non-dynamic class list of this component should go into the component-classes repo, right?

Yup!

File: index.html (hunk 1 above)
Comment (magnuh) [reply index.html:20]: Yup! 👍 Don't think that there was an alias... and not sure whether this link was the way to go forward, so just included it to get started. 😄

**Comment by magnuh** [comment]
> I don't think we want to change the API (e.g. `negative` -> `critical`) unless we have to, especially when it's something that is just a 'wording change' and isn't really meaningful IMO.
> 
> I don't know if this means the names need aligned back with design (e.g. these names would show up in Figma, and thus confusing developers because there's a difference) or if these are just token names no one but Adi will ever see.

I think the new names are better... but if that will be a problem for migrating or something else then it might be better to change it in the design as well... Less confusing to use the same words across all domains. I'll check with Adi. 👍

**Comment by magnuh** [comment]
> Yup! (I assume you mean they should have a color token for them we should be using)

The color tokens for the icons are already in place. 🙂  But I thought that the icons themselves could be different on all brands? Or are we going to use the same svg for all?

File: index.html (hunk 1 above)
Comment (magnuh) [reply index.html]: Alias in place now, so I updated this. 👍

File: index.html [hunk 2]
```diff
@@ -9,14 +9,7 @@
...
     />
-    <style>
-      f-docs-template {
-        visibility: hidden;
-      }
-      f-docs-template:defined {
-        visibility: visible;
-      }
-    </style>
+    <link rel="stylesheet" href="https://assets.finn.no/pkg/@warp-ds/tokens/v1/finn-no.css" />
```
Comment (magnuh) [inline index.html:12]: Should we use Finn as the base theme here? Or should there be like a `no-brand-theme.css`?

Comment (pearofducks) [reply index.html:12]: There should be a no-brand-theme used here, but I don't know if design has that done yet?

Comment (magnuh) [reply index.html:12]: We will go ahead with finn for now and change it later. 👍

Comment (Skadefryd) [reply index.html:12]: Prob easier to use the finn theme for now, since we know what that is supposed to look like.

**Review by AnnaRybkina — APPROVED** [review]

**Comment by BalbinaK** [comment]
[warpify react repo](https://trello.com/c/SwTwYlAH/136-warpify-react-repo)

**Pull request merged**